### ISSUE TITLE
Stack Overflow on save

### ISSUE THREAD

**Issue by RoyBatty64** [TES5Edit/TES5Edit#338]
The error isn't saved into the log, I managed to copy the message before it crashed to desktop. This is repeatable and happens every time with this plugin. I have tried loading and saving them both in GECK and there is no issues with either plugin. Seems isolated to dialogues. FNVEdit 3.1.2 version as the FO4Edit versions are not stable with FNV or FO3.

`[00:03] Copying [INFO:0602FEE3] ('No offense, kid, but you're not exactly the type I'm used to runnin' with.' in GRUP Topic Children of FollowersJerichoHire1a1 "You should come with me. I could use a hand." [DIAL:0002FEC1] for FollowersHireJericho "Hiring Jericho Quest" [QUST:0602FA33])
[00:03] Copying [INFO:0607FC11] ('No offense. You seem okay, kid. But your friend there, well... I'm used to working in smaller groups.' in GRUP Topic Children of FollowersJerichoHire1a1 "You should come with me. I could use a hand." [DIAL:0002FEC1] for FollowersHireJericho "Hiring Jericho Quest" [QUST:0602FA33])
[00:03] Copying done.
[00:04] Copying [INFO:000942A7] ('Sir, I regret to inform you that this unit cannot accept orders from an irresponsible officer!' in GRUP Topic Children of FollowersHired "I think we should travel together." [DIAL:00037084] for Followers "Quest to Handle Followers" [QUST:000371D1])
[00:04] Copying done.
[00:02] Copying [INFO:0606271F] ('It is regrettable, but I cannot follow one such as you. Restore your good name, and perhaps I shall reconsider.' in GRUP Topic Children of FollowersHired "I think we should travel together." [DIAL:00037084] for Followers "Quest to Handle Followers" [QUST:000371D1])
[00:02] Copying done.
[00:02] Copying [INFO:0607FC1C] ('Sorry man. I don't know what's gotten into you but you're just a little too intense for me. Maybe come back after you've chilled out and we'll see.' in GRUP Topic Children of FollowersHired "I think we should travel together." [DIAL:00037084] for Followers "Quest to Handle Followers" [QUST:000371D1])
[00:02] Copying done.
[00:02] Copying [INFO:0607FC1E] ('Sorry man. I don't know what's gotten into you but you're just a little too intense for me. Maybe come back after you've chilled out and we'll see.' in GRUP Topic Children of FollowersHired "I think we should travel together." [DIAL:00037084] for Followers "Quest to Handle Followers" [QUST:000371D1])
[00:02] Copying done.
[00:01] Copying [INFO:06096B89] ('My friend, I am saddened to hear that you have turned away from a noble path.' in GRUP Topic Children of FollowersHired "I think we should travel together." [DIAL:00037084] for Followers "Quest to Handle Followers" [QUST:000371D1])
[00:01] Copying done.
[00:04] Copying done.
[00:00] Saving: TTW Perks and Home Markers.esp.save.2016_02_24_13_15_02
[00:00] Error saving TTW Perks and Home Markers.esp.save.2016_02_24_13_15_02: Stack overflow
[00:00] Errors have occured. At least one file was not saved.
`

**Comment by zilav** [comment]
Have you tried to disable INFO sorting in Options?

**Comment by RoyBatty64** [comment]
That was it. Seems to be an ongoing issue, pls fix. :)

**Comment by zilav** [comment]
No way, that code is mindblowing :)
I hope that Elminster will come back and fix it, this is the only remaining issue after his previous fix.
If you can upload a test plugin that reproduces error, it would help a lot.

**Comment by RoyBatty64** [comment]
Is it possible you can make it use the exception handler so there is some notice to the user that a problem has occured?

I think it might be related to inserting NULL's in previous info topics where none exists in the source plugin. That can break things anyways under the right circumstances.

I can upload the plugin (but it's freely available on TTW site) that will cause this issue without fail, but as usual I can not make it happen with a bare bones plugin and it only happens with one that's complicated and has a lot of masters and overrides. This is only the second time I've had the issue, the other time was with FWE.

**Comment by zilav** [comment]
Exception is already kind of a notice that a problem occured. If it already happened, then no need to handle exception and show anything else since plugin's data is already invalid and can't be safely saved at this point.
Just leave a link to plugin here.

**Comment by InsanePlumber** [comment]
I once had similar problems with UUF3P.
It helped me Delete and generate a new settings file for xEdit.

**Comment by zilav** [comment]
It probably switched off INFO sorting (don't remeber it's default state), that's why it helped.

**Comment by eFrysTon** [comment]
Does this problem still exist?

**Comment by zilav** [comment]
Yes, sorting of INFOs doesn't work correctly sometimes, but I don't have reproducable test case unfortunately.
